osbs/api.py

```diff
@@ -491,6 +492,14 @@ def _check_labels(self, repo_info):
                 logger.error("required label missing from Dockerfile : %s",
                              labels.get_name(label))
 
+        try:
+            _, release_value = labels.get_name_and_value(utils.Labels.LABEL_TYPE_RELEASE)
+            if not RELEASE_LABEL_FORMAT.match(release_value):
+                logger.error("release label contains forbidden characters : %s", release_value)
+                raise OsbsValidationException("release label contains '/'")
```

Comment:
This error message is misleading. If the release label is `'abc'` user will get this message and be quite confused. Can we instead output the expected regex? `RELEASE_LABEL_FORMAT.pattern` might give you what you want.

Reply:
Yes... once you remove whitespace and comments. :-)
Perhaps `re.sub('\s', '', re.sub(r' #.*(\n|$)', '', RELEASE_LABEL_FORMAT.pattern))`

Reply:
sure